packages/stargate/src/signingstargateclient.ts

```diff
@@ -241,11 +252,43 @@ export class SigningStargateClient extends StargateClient {
   ): Promise<BroadcastTxResponse> {
     const withdrawMsg = {
       typeUrl: "/cosmos.distribution.v1beta1.MsgWithdrawDelegatorReward",
-      value: MsgWithdrawDelegatorReward.fromPartial({ delegatorAddress: delegatorAddress, validatorAddress }),
+      value: MsgWithdrawDelegatorReward.fromPartial({
+        delegatorAddress: delegatorAddress,
+        validatorAddress: validatorAddress,
+      }),
     };
     return this.signAndBroadcast(delegatorAddress, [withdrawMsg], this.fees.withdraw, memo);
   }
 
+  public async sendIbcTokens(
+    senderAddress: string,
+    recipientAddress: string,
+    transferAmount: Coin,
+    sourcePort: string,
+    sourceChannel: string,
+    timeoutHeight?: Height,
+    /** timeout in seconds */
+    timeoutTimestamp?: number,
```

Comment:
Should we make those required as `Height | undefined` and `number | undefined`? It seems to be dangerous to leave them out.

Reply:
The server will reject any packet that has both undefined. At least one must be set (it is valid to set both as well)

Reply:
Good. Let's force the user to explicitly set them to undefined instead of skipping the arguments.